.travis.yml

```diff
@@ -2,18 +2,22 @@ language: bash
 services: docker
 
 env:
-  - VERSION=0.12 PYTHON_VERSION=2.7 VARIANT=slim
-  - VERSION=0.12 PYTHON_VERSION=3.6 VARIANT=slim
-  - VERSION=0.12 PYTHON_VERSION=3.7 VARIANT=slim
-  - VERSION=0.12 PYTHON_VERSION=2.7 VARIANT=alpine
-  - VERSION=0.12 PYTHON_VERSION=3.6 VARIANT=alpine
-  - VERSION=0.12 PYTHON_VERSION=3.7 VARIANT=alpine
   - VERSION=1.0 PYTHON_VERSION=2.7 VARIANT=slim
```

Comment:
With Python 2.x being retired soon, I think that we should consider dropping 2.7 support from the matrix as well (for all versions and variants).